ui/panels-plugin/src/plugins/bar-chart/BarChartOptionsEditorSettings.tsx

```diff
@@ -65,6 +66,17 @@ export function BarChartOptionsEditorSettings(props: BarChartOptionsEditorProps)
     );
   };
 
+  const handleResetSettings: React.MouseEventHandler<HTMLButtonElement> = () => {
+    onChange(
+      produce(value, (draft: BarChartOptions) => {
+        draft.calculation = 'LastNumber';
```

Comment:
Looks like the DEFAULT_CALCULATION variable is set to 'Sum'

Which do you think is better default / reset value?

Reply:
Hmm looks like `DEFAULT_CALCULATION` won't actually ever be used. Stat chart and gauge chart use `LastNumber` as the default so I'm inclined to stick with that.

@eunicorn @sjcobb @juliepagano Any thoughts on what the default calculation should be?

Reply:
looks like Grafana also uses `LastNumber` as default so `LastNumber` sounds good to me

Reply:
+1 to that plan

Reply:
+1

Reply:
Thanks y'all, went with `LastNumber`! 

I also removed `export const DEFAULT_CALCULATION: CalculationType = 'Sum';` from ui/core/src/model/calculations.ts. It was referenced in the gauge and stat chart options editors but wasn't actually being used. Let me know if I'm missing something here and we need to keep it 496d16f563de0b3197a8839cffff9c2095801f32